doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -8,62 +8,157 @@ Building and Running the UFS Weather Model
-generic set of instructions in the ``README`` file at the top level of the `NCEPLIBS-external repository
-<https://github.com/NOAA-EMC/NCEPLIBS-external/tree/ufs-v2.0.0>`__ and at the top level of the `NCEPLIBS repository
-<https://github.com/NOAA-EMC/NCEPLIBS/tree/ufs-v2.0.0>`__. It may still be a good idea to look at some of the platform-
-and compiler-specific ``README`` files as a guide. Again, be sure to use the tag that corresponds to this version of the WM.
+   bacio/2.4.1
+   crtm/2.4.0
+   esmf/8.3.0b09
+   fms/2022.04
+   g2/3.4.5
+   g2tmpl/1.10.2
+   gftl-shared/v1.5.0
+   hdf5/1.10.6
+   ip/3.3.3
+   jasper/2.0.25
+   libpng/1.6.37
+   mapl/2.22.0-esmf-8.3.0b09
+   netcdf/4.7.4
+   pio/2.5.7
+   sp/2.3.3
+   w3emc/2.9.2
+   zlib/1.2.11
+
+The most updated list of common modules can be viewed in ``ufs_common.lua`` 
+`here <https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_common.lua>`__.
 
-The top-level ``README`` in the NCEPLIBS-external repository includes a troubleshooting section that may be helpful.
+.. attention::
+   Documentation is available for installing `HPC-Stack <https://hpc-stack.readthedocs.io/en/latest/>`__ 
+   and `spack-stack <https://spack-stack.readthedocs.io/en/latest/>`__, respectively. 
+   One of these software stacks (or the libraries they contain) must be installed before running the UFS Weather Model. 
+
+.. _GetData:
+
+============
+Get Data
+============
+
+The WM RTs require input files to run. 
+These include static datasets, files that depend on grid resolution and 
+initial/boundary conditions, and model configuration files. On Level 1 systems, 
+the data required to run the WM RTs are already available in the following locations: 
+
+.. _DataLocations:
+.. table:: Data Locations for Level 1 Systems
+
+   +--------------+-----------------------------------------------------+
+   | Machine      | File location                                       |
+   +==============+=====================================================+
+   | Cheyenne     | /glade/scratch/epicufsrt/GMTB/ufs-weather-model/RT  |
+   +--------------+-----------------------------------------------------+
+   | Gaea         | /lustre/f2/pdata/ncep_shared/emc.nemspara/RT        |
+   +--------------+-----------------------------------------------------+
+   | Hera         | /scratch1/NCEPDEV/nems/emc.nemspara/RT              |
+   +--------------+-----------------------------------------------------+
+   | Jet          | /mnt/lfs4/HFIP/hfv3gfs/role.epic/RT                 |
+   +--------------+-----------------------------------------------------+
+   | Orion        | /work/noaa/nems/emc.nemspara/RT                     |
+   +--------------+-----------------------------------------------------+
+   | WCOSS2       | /lfs/h2/emc/nems/noscrub/emc.nems/RT                |
+   +--------------+-----------------------------------------------------+ 
+
```

Comment:
S4's data is located in `/data/prod/emc.nemspara/RT`.

Reply:
@DavidHuber-NOAA Can we confirm s4 baseline and input files are maintained at /data/prod/emc.nemspara/RT ? If regularly maintained there, I agree with adding S4 to the list level 2 platform?

Reply:
@jkbk2004 Yes, the baseline and input files are maintained there.  An rsync is performed nightly from Hera to keep them up to date.  Regression tests are also performed at least once monthly.

Reply:
Ok, added some S4 info here and in Ch 3! :)